pkg/updater/gcs_test.go

```diff
@@ -1828,8 +1828,67 @@ func TestConvertResult(t *testing.T) {
 				},
 			},
 		},
+		{
+			name: "Ginkgo V2 skipped with message",
+			nameCfg: nameConfig{
+				format: "%s",
+				parts:  []string{testsName},
+			},
+			result: gcsResult{
+				started: gcs.Started{
+					Started: metadata.Started{
+						Timestamp: now,
+					},
+				},
+				finished: gcs.Finished{
+					Finished: metadata.Finished{
+						Timestamp: pint(now + 1),
+						Passed:    &yes,
+					},
+				},
+				suites: []gcs.SuitesMeta{
+					{
+						Suites: &junit.Suites{
+							Suites: []junit.Suite{
+								{
+									Results: []junit.Result{
+										{
+											Name:    "visible skip non-default msg",
+											Skipped: &junit.Skipped{Message: *pstr("non-default message")},
+										},
+										{
+											Name:    "invisible skip default msg",
+											Skipped: &junit.Skipped{Message: *pstr("skipped")},
+										},
+										{
+											Name:    "invisible skip msg empty",
+											Skipped: &junit.Skipped{Message: *pstr("")},
```

Comment:
this kind of should be tested for v1, we won't hit this situation in v2 afaik, add this to increase the test coverage.